index.html

```diff
@@ -0,0 +1,555 @@
+                </ul>
+            </div>  
+        </nav>
+        <main>
+            <div class="fixed-nav">
+                <div class="bottom-nav">
+                    <div class="tnw-logo">
+                        <img alt="TNW-logo" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg">
+                    </div>
+                    <ul>
+                       <li><a href="#">Latest</a></li>
+                       <li><a href="#">Hard Fork</a></li>
+                       <li><a href="#">Plugged</a></li>
+                       <li><a href="#">Fundamental</a></li>
+                       <li><a href="#">ReadMe</a></li>
+                       <li><a href="#">Growth Quarters</a></li>
+                       <li><a href="#">Neural</a></li>
+                    </ul>
+                    <div class="border-icon">
+                        <div class="icon-border1"><i class="search icon"></i></div>
+                        <div class="icon-border2"><i class="bars icon"></i></div>
+                    </div>
+                      
+               </div>
+               <p class="p1"><a href="#">Did you know we have a TECH FESTIVAL? Check it out <i class="long arrow alternate right icon"></i></a></p>
+           </div>
+            
+            <div class="main-grid">
+                <div class="tnw">
+                    <div>
+                        <img src="./images/tnw-w.png" alt="tnw-logo"/>
+                        <time datetime="2020-03-13 20:00">Friday - March 13, 2020</time>
+                    </div> 
+                    <article>
+                        <h3>TECH</h3>
+                        <h2>How to handle toxic people as an<br> online community manager</h2>
+                        <div class="time-author">
+                          <p>by Milena Abrosimova</p>
+                          <p><i class="clock outline icon"></i>50 minutes ago</p>
+                          <p><i class="share alternate icon"></i>0</p>
+                        </div>
+                    </article>
+                </div>
+                <div class="reddit">
+                  <article>
+                      <h2>Reddit, Paypal, Spotify and more <br> are speaking at TNW's tech festival in Amsterdam</h2>
+                  </article>  
+                </div>
+                <div class="microsoft">
+                    <article>
+                        <h2>Satoshi Nakaboto: 'Bitcoin drops <br> a jaw-dropping 37% in 24 hours'</h2>
+                        <div class="time-author">
+                          <p>by Satoshi Nakaboto</p>
+                          <p><i class="clock outline icon"></i>2 hrs ago</p>
+                          <p><i class="share alternate icon"></i>5</p>
+                        </div>
+                    </article>
+                </div>
+
+                <section>
+                    <h2>Latest New <i class="chevron right icon"></i></h2>
```

Comment:
- [ ] The latest news section title should be aligned with the articles' images (both medium and large screens) and should be red.
![Screenshot from 2020-03-18 12-01-19](https://user-images.githubusercontent.com/51926937/76992109-54b89e80-6910-11ea-8cb0-09eacf773417.png)

Original:
![Screenshot from 2020-03-18 12-01-33](https://user-images.githubusercontent.com/51926937/76992126-5aae7f80-6910-11ea-8bc2-1d1484b28087.png)

- [ ] In the mobile version, your page has larger images. Please fix them.
![Screenshot from 2020-03-18 12-07-13](https://user-images.githubusercontent.com/51926937/76992582-166faf00-6911-11ea-81e5-3dd7cc127ea4.png)

Original:
![Screenshot from 2020-03-18 12-07-06](https://user-images.githubusercontent.com/51926937/76992591-1a033600-6911-11ea-8673-da9a03292a0e.png)